nodejs/eks/nodes/nodegroup.ts

```diff
@@ -1933,6 +1953,39 @@ export function createManagedNodeGroup(
         });
     }
 
+    // Create a placement group if EFA support is enabled
+    const createPlacementGroup = args.enableEfaSupport === true;
+    const placementGroup = createPlacementGroup
```

Comment:
Can this just be 

```ts
const placementGroup = args.enableEfaSupport ? new aws.ec2.PlacementGroup() : undefined;
```

Reply:
It could, but I wanted to make the whole placementGroup logic more explicit (we're also using the `createPlacementGroup` const further down).

But I'm also happy to change it if it's more understandable that way!